packages/cli/src/commands/new/project.ts

```diff
@@ -44,6 +44,11 @@ export default class Project extends Command {
       description:
         'package name implementing the cloud provider integration where the application will be deployed (i.e: "@boostercloud/framework-provider-aws"',
     }),
+    default: flags.boolean({
+      char: 'd',
+      description: 'generates the project with default parameters (i.e. --license=MIT)',
+      default: true,
```

Comment:
Shouldn't this be `default: false`? If I recall correctly, `default` makes the flag enabled by default if it is not specified, which is not what we need

Reply:
Whoops you're right!!